components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/core/Constants.java

```diff
@@ -34,6 +34,8 @@
         // Client errors.
         ERROR_CODE_INVALID_APPLICATION_ID("10001", "Invalid application ID",
                 "An application with ID: %s cannot be found for user: %s."),
+        ERROR_CODE_APPLICATION_NOT_FOUND("10002", "Invalid application ID.", "Unable to find any " +
+                "application with the application ID: %s."),
```

Comment:
Can we add the tenant as well as a param?

Reply:
Addressed in [892099c](https://github.com/wso2/identity-api-user/pull/129/commits/892099c6c5888c3333cacd8be25ef6a8adccda93)